specs/kvstore/KeyValueStore.fizz

```diff
@@ -2,9 +2,14 @@
 deadlock_detection: false
 options:
     max_actions: 100
+    crash_on_yield: false
 action_options:
     SlateDb.Put:
         max_actions: 2
```

Comment:
Nit: I can't find any docs on fizzbee.io about what this is. My search just finds log lines with it. I tried the on-site search, too.

![Screenshot 2024-09-23 at 4 08 04 PM](https://github.com/user-attachments/assets/aab7de92-dc1b-4b18-b89b-59c21bbbc7e3)

Reply:
These configurations are documented here https://fizzbee.io/tutorials/frontmatter/#max-actions
(The first link on the google search)
The configuration is specified as YAML config. It can be added as a separate file fizz.yaml on the same directory or as a front matter at the top of the file.

`max_actions` limits the maximum number of times an action can be triggered on a single path (timeline). This is usually required to limit the state space. For example in this case, in the worst case, each Put will create one WAL file. Even if we limit the key space for example, only a single possible key and single possible value, we will have infinite possible states. So we need a way to limit the number of states, here limiting the Puts seem to be the natural choice. 

I need to update the documentation with the new options added (`crash_on_yield`) and the format for role actions.